zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/SchemaManager.java

```diff
@@ -55,11 +56,12 @@ public void startup() {
 
     //  used to update existing indices/templates
     LOG.info("Update index schema. '{}' indices need to be updated", newIndexProperties.size());
-    updateSchema(newIndexProperties);
+    updateSchemaMappings(newIndexProperties);
     LOG.info(
         "Update index template schema. '{}' index templates need to be updated",
         newIndexProperties.size());
```

Comment:
Not related to your change, but I think we want to log `newIndexTemplateProperties.size()` here, or?